test/integration/Sanity.e2e.test.js

```diff
@@ -32,4 +32,85 @@ describe('Annotations', () => {
         // Assert that at least one annotation is present on the image
         cy.get('[data-testid^="ba-AnnotationTarget"]');
     });
+
+    describe('discoverability', () => {
+        it('should be able to draw a region without entering the mode', () => {
+            // Show the preview
+            cy.showPreview(Cypress.env('FILE_ID_DOC_SANITY'));
+
+            // Wait for the region layer to be present
+            cy.getByTestId('ba-Layer--region');
+
+            // Add a region annotation on the document
+            cy.drawRegion();
+            cy.getByTestId('ba-Popup-cancel').click();
+
+            // Assert that annotation target is not active
+            cy.get('.ba-RegionAnnotation').should('not.have.class', 'is-active');
+
+            // Select annotation target
+            cy.get('.ba-RegionAnnotation').click();
+
+            // Assert that annotation target is active
+            cy.get('.ba-RegionAnnotation').should('have.class', 'is-active');
+        });
+
+        it('should be able to promote a highlight without entering the mode', () => {
+            // Show the preview
+            cy.showPreview(Cypress.env('FILE_ID_DOC_SANITY'));
+
+            // Wait for the region layer to be present
+            cy.getByTestId('ba-Layer--region');
+
+            // Add a highlight annotation on the document
+            cy.selectText();
+            cy.getByTestId('ba-PopupHighlight-button').click();
+            cy.getByTestId('ba-Popup-cancel').click();
+
+            // Assert that annotation target is not active
+            cy.get('.ba-RegionAnnotation').should('not.have.class', 'is-active');
+
+            // Select annotation target
+            cy.get('.ba-RegionAnnotation').click();
+
+            // Assert that annotation target is active
+            cy.get('.ba-RegionAnnotation').should('have.class', 'is-active');
+        });
+
+        it('should leave region mode if zooming in on an image', () => {
+            // Show the preview
+            cy.showPreview(Cypress.env('FILE_ID_IMAGE_SANITY'));
+
+            // Wait for viewer to load
+            cy.get('.bp').should('have.class', 'bp-loaded');
```

Comment:
Do we need to check both of these classes? We should try to limit how much we rely on the specific structure of box-content-preview, since it creates a dependency in our tests.